.github/workflows/reusable-unit.yml

```diff
@@ -110,7 +110,7 @@ jobs:
         run: yarn ${{ matrix.package }} build
 
       - name: Check ${{ matrix.package }} bundle size
-        if: ${{ matrix.package == 'react' }}
+        if: ${{ matrix.package == 'react' || matrix.package == 'react-liveness' }}
         run: yarn react size
```

Comment:
My understanding of this file is rudimentary at best, but won't `yarn react size` only run against `ui-react`? Or maybe that is intentional?

Reply:
Ah my mistake, didn't read this closely enough